types/testing-library__jest-dom/index.d.ts

```diff
@@ -0,0 +1,35 @@
+// Type definitions for @testing-library/jest-dom 4.0
+// Project: https://github.com/testing-library/jest-dom
+// Definitions by: Damien Senger <https://github.com/hiwelo>
+//                 Łukasz Fiszer <https://github.com/lukaszfiszer>
+//                 Revath S Kumar <https://github.com/revathskumar>
+//                 Ernesto García <https://github.com/gnapse>
+//                 Weyert de Boer <https://github.com/weyert>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 3.0
```

Comment:
According to https://github.com/testing-library/jest-dom#usage this you need to have `extend-expect` to be present that contains the globals

Reply:
Yes, as this library is extending expect() to have more matchers.

Reply:
What I meant was there needs to be a file `extend-expect.d.ts` that defines all the global declaraitons.